src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -204,6 +202,9 @@ def export_model(self, model_path, mask_path=None, onnx_path=None, input_shape=N
         input_shape : list or tuple
             input shape to onnx model
         """
+        if self.detect_modules_to_compress() and not self.mask_dict:
+            _logger.warning('You may not use self.mask_dict in base Pruner class to record masks')
+            print('Warning: You may not use self.mask_dict in base Pruner class to record masks')
```

Comment:
Please remove the print.
Maybe we should add logger configuration code for "standalone mode" now.

Reply:
any proposal?

Reply:
removed, and add a new issue to track this log refactor